### DISCUSSION TITLE
MvvmCross.Plugin.Location Source code not available

### DISCUSSION THREAD

**Discussion by viswanath-pamarthi** [MvvmCross/MvvmCross#4454] [Q&A] [answered]
Anyone have idea on where I can find the source code of the plugIn - "MvvmCross.Plugin.Location" , I looked through source code of MvvmCross library and couldn't find it, please find.

**Comment by Cheesebaron** [comment, marked as answer]
Select the appropriate tag in the branches/tags dropdown. Some of the plugins where recently removed because of incompatibilities with .net6.0

**Reply by viswanath-pamarthi** [reply]
Thanks, I found it, I see the plugin code is totally removed in your latest commit for .Net 6 support:
https://github.com/MvvmCross/MvvmCross/tree/4a710f6ea5533fe0e9e2920b15dc28662c4786a2/MvvmCross.Plugins/Location
